### DISCUSSION TITLE
vue-materialize-datatable theme customization

### DISCUSSION THREAD

**Discussion by ValentineSean** [MicroDroid/vue-materialize-datatable#73] [Q&A]
How can I change:
-table headers <th> font family, font size, font color and other styles.
-table responsiveness especially making it horizontal scrollable when the screen width can not contain all columns.

Also when exporting excel after filtering, the excel sheet is populated with all data not only filtered data. But export as PDF is giving only filtered data.

**Comment by obas101** [comment]
Hi any fix for horizontal scroll for table responsiveness?

**Reply by MicroDroid** [reply]
no

**Reply by MicroDroid** [reply]
I almost forgot this library exists
